mindfiredigital/PivotHead
# Bug/12 code clean up

# Pull Request Description

This pull request addresses the following issue:

**Feature**: [GitHub Issue #12  ](https://github.com/lakinmindfire/PivotHead/issues/12)

### Changes Made:
- node_module or some unneccessary file are commited like .pnpm-lock file, build file etc .
- pnpm-lock is necessary file for consitent development 
- Update gitignore to ignore  build created inside the packages as well.

### Checklist:
- [x]  node_module or some unneccessary file are commited like .pnpm-lock file, build file etc .
- [x ] pnpm-lock is necessary file for consitent development 
- [x]  Update gitignore to ingore for build created inside the packages as well.
- [x] The file(s) or folder(s) now contain changes as specified in the issue I worked on.
- [x] I certify that I ran the checklist.


### Testing steps :-
1. Install pnpm globally
  `npm i -g pnpm`

2. Install dependency at root.

  `pnpm install`

3. Build package at root

`pnpm run build`

4. Go inside the folder `examples/vanilla-js-demo`

`cd examples/vanilla-js-demo`

5. Then do install `pnpm install` and then run `pnpm start`. Go to local url to see the package demo.
6. If everything is ok. All the above steps will run correctly.



### Screenshot

Not required

### Preview

Not required

---

### Ping:
Please review this pull request.
